front/public/locales/fr/stdcm.json

```diff
@@ -0,0 +1,3 @@
+{
+  "searchFailed": "Echec de la recherche de sillon"
+}
```

Comment:
What is the purpose of this file ? You don't seem to use it. Is it for later ?

Reply:
Removed